<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherdashboard</class>
 <widget class="QMainWindow" name="teacherdashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="HEADERFRAME">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #6A1B9A;
background-color: rgb(130, 91, 154);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>19</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
color: rgb(0, 255, 255);</string>
         </property>
         <property name="text">
          <string>TuiPath</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="td_dashboard">
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="td_createclass">
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>CREATE CLASS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="td_classes">
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Classes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="td_attendance">
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string> Attendance</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="td_payments">
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>payments</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="td_settings">
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
color: rgb(0, 0, 0);

background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="td_profile">
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
image: url(:/images/profile.png);
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logothutton">
         <property name="font">
          <font>
           <family>Algerian</family>
           <pointsize>12</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
image: url(:/images/logout.png);
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
